src/main/kotlin/com/devfloor/untitled/common/presentation/CommonExceptionAdvice.kt

```diff
@@ -0,0 +1,15 @@
+package com.devfloor.untitled.common.presentation
+
+import com.devfloor.untitled.common.exception.EntityNotFoundException
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.ResponseStatus
+
+@ControllerAdvice
+@ResponseStatus(HttpStatus.NOT_FOUND)
+class CommonExceptionAdvice {
+    @ExceptionHandler(value = [EntityNotFoundException::class])
+    fun handleNotFoundException(errorMessage: String): String = errorMessage
+
+}
```

Comment:
ResponseStatus 위치가 잘못된 듯? 그리고 맨 마지막에 빈칸 추가됐어! 그리고 표현식이니까 반환 타입 제거! 메서드명도 수정!

```suggestion
class CommonExceptionAdvice {
    @ExceptionHandler(value = [EntityNotFoundException::class])
    @ResponseStatus(value = HttpStatus.NOT_FOUND)
    fun handleEntityNotFoundException(errorMessage: String) = errorMessage
}
```

Reply:
반영완료 !